src/main/java/openblocks/common/entity/EntityHangGlider.java

```diff
@@ -230,7 +230,7 @@ public double getNoise() {
 
 		if (player.dimension != 0)
 			noise = 0;
-		else if (worldObj.isRaining() && !strong)
+		else if (worldObj.isRaining() && player.isWet() && !strong)
```

Comment:
:partying_face:
Thanks a lot for fixing. I always wanted it to act like that, yet I was not aware of isWet() (and didn't want to implement an expensive solution).

Reply:
Feel free to take anything of course, but I accidentally opened this against the upstream 😋